src/controllers/tripsController.js

```diff
@@ -428,4 +429,40 @@ export default class requestController {
       return Response.errorResponse(res, 500, res.__('server error'));
     }
   }
+
+  /**
+   * @description this function provides requester's trips statistics
+   * @param  {object} req
+   * @param  {object} res
+   * @return {object} number of all created trips
+   */
+  static async requesterTripStats(req, res) {
+    const { user } = req;
+    try {
+      const allTrips = await db.Request.findAll({
+        where: { userId: user.id },
+      });
+      StatsController.stats(allTrips, res);
+    } catch (err) {
+      return Response.errorResponse(res, 500, res.__('server error'));
+    }
+  }
+
+  /**
+   * @description this function provides manager's trips statistics
+   * @param  {object} req
+   * @param  {object} res
+   * @return {object} number of all created trips by his/her direct report
+   */
+  static async managerTripStats(req, res) {
+    const { user } = req;
+    try {
+      const allTrips = await db.Request.findAll({
+        where: { managerId: user.id },
+      });
```

Comment:
does the manager see stats for individual requester or all the requester

Reply:
the manager sees stats of all requests made not for an individual.